src/__tests__/keyBindingsForTest.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+KEY_BINDINGS_FOR_TEST_CONFIG_CONTENT = "[bindings]\nr = rspec\ns = subl\n"
```

Comment:
ah interesting so you use this in the testing below but you define it here? guess thats nice to keep it in sync with the mock